packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -306,6 +308,7 @@ class EditableText extends StatefulWidget {
        assert(focusNode != null),
        assert(obscureText != null),
        assert(autocorrect != null),
+       assert(readOnly != null),
```

Comment:
also `assert(showCursor != null)`?

Reply:
I changed that so showcursor can be null, if it is null it will based be set to !readOnly